pkg/proc/interface.go

```diff
@@ -134,3 +139,18 @@ func (cctx *ContinueOnceContext) GetManualStopRequested() bool {
 	defer cctx.StopMu.Unlock()
 	return cctx.manualStopRequested
 }
+
+// NumProcs returns the number of processes in the target group being resumed.
+func (cctx *ContinueOnceContext) NumProcs() int {
+	return len(cctx.grp.targets)
+}
+
+// Proc returns the i-th process in the group.
+func (cctx *ContinueOnceContext) Proc(i int) ProcessInternal {
+	return cctx.grp.targets[i].proc
+}
+
+// AddTarget adds a target to the group being resumed.
+func (cctx *ContinueOnceContext) AddTarget(tgt *Target) {
```

Comment:
This also feels like we should just be calling this directly on `TargetGroup`.

Reply:
Same as above. It would be really bad if someone saw the AddGroup method on TargetGroup and though they could call NewTarget and pass it to (*TargetGroup).AddTarget.